platform/ios/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 Mapbox welcomes participation and contributions from everyone. Please read [CONTRIBUTING.md](../../CONTRIBUTING.md) to get started.
 
+## 4.11.0
```

Comment:
Since `release-liquid` is 4.10.0, its changelog shouldn’t include anything to do with 4.11.x.